ProboCI/probo-reaper
# Feature/lol/database backup endpoint

We want to be able to export and backup the leveldb database used
by the reaper. To do this easily, we create a command line
endpoint that can be used to directly stream the contents of
leveldb to a json file.

This is going to be used by Jenkins to back up our reaper data.
### To Test
- `sudo ./bin/probo-reaper db-export -c /etc/probo/reaper.yaml > backup.json`
- Ensure the json dump is properly formatted.
